### ISSUE TITLE
Drag-from-widget to add next widget constantly produces error messages

### ISSUE THREAD

**Issue by wvdvegte** [biolab/orange3#6410]
**What's wrong?**
Since recently, I can no longer easily drag a connector from a widget and select a next widget in the contextual menu. The first time directly after dragging, an Unexpected Error window comes up, with a report that can be submitted. Next, a larger error message window pops up. If I click Ignore, I can finally select the next widget. Every next time I want to add new widgets this way, two of these large-size error message windows pop up consecutively. After clicking Ignore on each, again I can proceed.

It appears this started happening since (1) I updated Mac OS to Ventura 13.3.1 and (2) I updated the Text mining add-on to v. 1.12.2

**How can we reproduce the problem?** 
See attached movie:
[Screen Recording 2023-04-13 at 15.42.50.mov.zip](https://github.com/biolab/orange3/files/11222910/Screen.Recording.2023-04-13.at.15.42.50.mov.zip)

**What's your environment?**

- Operating system: Mac OS to Ventura 13.3.1 (Silicon)
- Orange version: 3.34.1
- How you installed Orange: from DMG

_edit_: updated add-on was text mining, not timeseries.

**Comment by janezd** [comment]
<img width="756" alt="Screenshot 2023-04-14 at 15 19 43" src="https://user-images.githubusercontent.com/2387315/232055100-c68f2aae-b216-4c88-9712-bf13b5a357c9.png">

Does updating the text mining add-on resolve the problem?

It seems that a part of text mining add-on expects a different version of numpy - which came from whoever knows - and fails at import. Does anyone know what's going on? @markotoplak, @PrimozGodec?

This resurrects the topic of pinning: Orange gives no warning if an add-on does not support a recent version of some library. One add-on (or Orange itself) may update pandas and an add-on may crash until the user updates it.

@ales-erjavec, beyond this, a crashing add-on can cause general problems in canvas. Could it be made more resilient?

@wvdvegte, thanks for this report. The concrete problem itself may be trivial to solve, but you warned us about a nasty general problem.

**Comment by wvdvegte** [comment]
Hm, I confused text mining and timeseries, indeed. I uninstalled text mining, then reinstalled it. That did not help, unfortunately.

**Comment by wvdvegte** [comment]
Update: I didn't discover at first, but I completely lost the text add-on after trying to reinstall it after uninstalling it as reported above. So far, I haven't been successful in getting it back.
I will open a separate issue for it.

**Comment by PrimozGodec** [comment]
@wvdvegte, sorry that it took us long to review your error. I tried to reproduce the error but could not (I have the same version of MacOS, Orange, and Text addon). Can you report to us the versions of the Python packages that you have installed? 

You get versions by opening the Terminal and typing `/Applications/Orange3.app/Contents/MacOS/pip freeze` if you have Orange installed in the Applications folder. Could you change the path if your Orange is installed somewhere else?

**Comment by wvdvegte** [comment]
@PrimozGodec "unfortunately", the problem disappeared. After uninstalling the text add-on as suggested above, and not being able to get it back, I uninstalled Orange and all plugins, then reinstalled everything as reported in #6416. This also eliminated the  drag-from-widget problem. My apologies for not reporting accordingly!

**Comment by PrimozGodec** [comment]
@wvdvegte, thank you for reporting and helping us. We spotted the error. It turned out that Gensim (one of the Text's dependencies) made a mistake in building the package with too new Numpy in version 4.3.1. We temporarily fixed it by omitting the problematic version https://github.com/biolab/orange3-text/pull/967. I also proposed the PR that will fix the problem in their repository's next version of Gensim.
